.travis.yml

```diff
@@ -4,10 +4,11 @@ php:
   - 5.4
   - 5.5
   - 5.6
+  - 7.0
   - hhvm
 
 env:
-  - SYMFONY_VERSION="2.7.*,>2.7.6"
+  - SYMFONY_VERSION="2.8.*"
```

Comment:
I would default to what composer resolves, instead of locking it to a version. So at the moment, that'll be 3.0.